dependencies.yaml

```
@@ -31,3 +31,6 @@ websocketpp:
 libevse-security:
   git: https://github.com/EVerest/libevse-security.git
   git_tag: v0.3.0
+libwebsockets:
+  git: https://github.com/warmcat/libwebsockets.git
+  git_tag: v4.3.3
```

```suggestion
  git_tag: v4.3.3
  
```
(Newline at end of file)